shared/teams/team/member/add-to-channels.tsx

```diff
@@ -43,12 +44,17 @@ const AddToChannels = (props: Props) => {
     : channelInfos
   const items = [
     ...(filtering ? [] : [{type: 'header' as const}]),
-    ...channels.map(c => ({
-      channelname: c.channelname,
-      conversationIDKey: c.conversationIDKey,
-      numMembers: c.numParticipants,
-      type: 'channel' as const,
-    })),
+    ...channels.map(c => {
+      const participantInfo = Container.useSelector(s =>
+        ChatConstants.getParticipantInfo(s, c.conversationIDKey)
```

Comment:
I think this is at risk of crashing if the number of channels changes. Can we consolidate this into the selector that is getting `channelInfos`?